### DISCUSSION TITLE
context_window_size hard coded to 1000 in prompts.py

### DISCUSSION THREAD

**Discussion by JEHollierJr** [llmware-ai/llmware#260] [General]
I see the following to set the size of the context window in prompts.py

Line 62
        # default batch size, assuming all LLMs have min 2048 full context (50% in / 50% out)
        self.context_window_size = 1000

can this be made configurable or set to the max of the models?  

I see some of the newer models have a larger context window that would be useful to be able to access.

**Comment by NYDocutest** [comment]
Hi This number is actually over-ridden automatically based on the actual number of tokens in the context window allowed by the individual models themselves (50% input/50% output).
